java-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiProperties.java

```diff
@@ -0,0 +1,83 @@
+package com.yoti.api.spring;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Allows properties for Yoti configuration to be supplied via spring properties (e.g. YAML or properties file).
+ */
+@ConfigurationProperties(prefix = "com.yoti")
+public class YotiProperties {
+
+    /**
+     * The Yoti Dashboard Application ID.
```

Comment:
I might misunderstand, but this is actually the dashboard application ID ant not the SDK one. The SDK one is already configured in the class https://github.com/getyoti/java/blob/master/java-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java#L14 which existed before this change. This change intends to allow developers to configure their connect application id in their app so that they can do things like forward their login buttons to the correct Yoti Connect login screen. Let me know if you need more detail on this, I can talk you through the code sample in the spring example module that uses both SDK ID and dashboard application id.

Reply:
Sorry, you are right :- )